chapters/data-analysis.tex

```diff
@@ -1,292 +1,488 @@
+other non-responses into extended missing values.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Cleaning\#Survey\_Codes\_and\_Missing\_Values}}
+String variables need to be encoded, and open-ended responses, categorized or dropped\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Cleaning\#Strings}}
+(unless you are using qualitative or classification analyses, which are less common).
+Finally, any additional information collected only for quality monitoring purposes,
+such as notes and duration fields, can also be dropped.
+
+% Outputs -----------------------------------------------------------------
+
+% Data set
+The most important output of data cleaning is the cleaned data set. 
+It should contain the same information as the raw data set,
+with no changes to data points.
+It should also be easily traced back to the survey instrument,
+and be accompanied by a dictionary or codebook.
+Typically, one cleaned data set will be created for each data source,
+i.e. per survey instrument.
+Each row in the cleaned data set represents one survey entry or unit of observation.\sidenote{\cite{tidy-data}}
+If the raw data set is very large, or the survey instrument is very complex,
+you may want to break the data cleaning into sub-steps, 
+and create intermediate cleaned data sets
+(for example, one per survey module).
+Breaking cleaned data sets into the smallest unit of observation inside a roster
+make the cleaning faster and the data easier to handle during construction.
+But having a single cleaned data set will help you with sharing and publishing the data.
+To make sure this file doesn't get too big to be handled,
+use commands such as \texttt{compress} in Stata to make sure the data
+is always stored in the most efficient format.
+
+% Documentation
+Throughout the data cleaning process, you will need inputs from the field, 
+including enumerator manuals, survey instruments, 
+supervisor notes, and data quality monitoring reports.
+These materials are part of what we call data documentation
+\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Documentation}}
+\index{Documentation},
+and should be stored in the corresponding folder, 
+as you will probably need them during analysis and publication.
+Include in the \texttt{Documentation} folder records of any
+corrections made to the data, including to duplicated entries,
+as well as communications from the field where theses issues are reported.
+Make sure to also have a record of potentially problematic patterns you noticed
+while exploring the data, such as outliers and variables with many missing values.
+Be very careful not to include sensitive information in 
+documentation that is not securely stored, 
+or that you intend to release as part of a replication package or data publication.
+
+\section{Indicator construction}
 
+% What is construction -------------------------------------
+Any changes to the original data set happen during construction.
```

Comment:
_changes_ is too vague here. It might be read as we make arbitrary changes to the original data. We should call it _corrections_ or something that makes it sound more formal.

Reply:
> Data construction is the process of interpreting the data points as provided in the raw data to make them suitable for analysis.

Reply:
Changed line 261 to Ben's suggestion in fe5b309